### PR TITLE
Performance cleanup

### PR DESCRIPTION
**Pull request by pyrello**
Related to #7683

# How to test
- For each of the standard list block types, ensure that limit, limit per page, and paging is still working as expected and hasn't changed for existing blocks.
- Ensure that breadcrumb and caption are still displaying for featured images when they are displayed on a full node.

## Query logging
```
ddev general_log
```
Visit a page locally or run a command (in a different terminal window) and observe the overwhelming amount of query logs. Cmd-c to stop logging.

### CODE REVIEW

**Comment by joewhitsitt** [comment]
pulling to test. should we extend this to the events listblock?

**Comment by joewhitsitt** [comment]
Looks like we are setting a default in code for items per page rather than config

https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/layout_builder_custom/layout_builder_custom.module#L301

>// This case should fall through to the next
>// to set the default items_per_page.

I don't think the fall through switch statement setup here is working.

File: config/default/views.view.people_list_block.yml [hunk 1]
```diff
@@ -687,19 +687,19 @@ display:
...
           total_pages: null
           id: 0
           tags:
             next: 'Next ›'
             previous: '‹ Previous'
           expose:
-            items_per_page: true
+            items_per_page: false
             items_per_page_label: 'Items per page'
             items_per_page_options: '9'
```
Comment (joewhitsitt) [inline config/default/views.view.people_list_block.yml]:
```suggestion
            items_per_page_options: '10'
```

**Review by joewhitsitt — APPROVED** [review]
I'd be curious to know what can be done with the general_log outputting like that. My terminal was overwhelmed and put a limit, not letting me scroll back to the beginning of the command. That was just one page load (sandbox home).

Though I think the config is good to keep similar here, it is the defaults set in code that control the items_per_page.

Article caption/breadcrumb/featured image still works.

**Comment by joewhitsitt** [comment]
deleted caches and restarted the build

**Comment by pyrello** [comment]
> I'd be curious to know what can be done with the general_log outputting like that. My terminal was overwhelmed and put a limit, not letting me scroll back to the beginning of the command. That was just one page load (sandbox home).

I had to switch to Apple's terminal to be able to see all the output from the command. PhpStorm's terminal has a maximum output that gets truncated after its exceeded. My thought is that it would be even better to use the Console app or some other log viewer that could allow you to filter results more easily.

> Though I think the config is good to keep similar here, it is the defaults set in code that control the items_per_page.

The views config changes help with the potential confusion of visiting the Views UI and seeing different results there than the default block settings. It will never actually have an effect on the block output.

**Pull request merged**